next/pages/_app.tsx

```diff
@@ -14,6 +15,10 @@ import { QueryParamProvider } from 'use-query-params'
 import ContentImage from '../components/atoms/ContentImage'
 import { HomepageMarkdown } from '../components/atoms/HomepageMarkdown'
 
+const DynamicChat = dynamic(() => import('./chat'), {
```

Comment:
Please explain why this approach.
If simple isClient works, please use that. https://usehooks-ts.com/react-hook/use-is-client